src/components/LandingPage/Testmonials.js

```diff
@@ -107,10 +107,12 @@ const Testmonials = () => {
           }
           name={"Tiana"}
         />
-        <TestimonialButton
-          buttonText={"Join our program"}
-          className={classes.button}
-        />
+        <a href="how-it-works">
```

Comment:
Instead of putting an <a> tag for Testinmonial Button, wrap the actual testinmonial button: 
```suggestion
        
export default function TestimonialButton({buttonText, link}) {
  const classes = useStyles()


  return (
  <a href={link}>
    <button className={classes.testimonialButton} >
      {buttonText}
    </button>
  </a>
```

Reply:
Done!